src/components/Map.js

```diff
@@ -323,6 +321,10 @@ class MapComponent extends Component {
     return markers;
   }
 
+  tileHasVectors(tile) {
+    return tile.type === "map" && tile.assetTypes && tile.assetTypes.length > 0;
```

Comment:
Ahh ok, het staat hiet al. tile.type is hier altijd map, we zitten in de Map tile.